recipes/bg-mpl-stylesheets/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set group = "Billingegroup" %}
+{% set name = "bg-mpl-stylesheets" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "c501bf584feb46f792c3dbf515ebb2ae97da2cfdf07baad1cbd952baf4a52720" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/{{ group }}/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib-base
+
+test:
+  imports:
+    - bg_mpl_stylesheet
+
+about:
+  home: https://github.com/{{ group }}/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Defines a standardized matplotlib stylesheet for the Billinge Group'
+
+extra:
+  recipe-maintainers:
+    - sbillinge
+    - Billingegroup
```

Comment:
This is an org, this thus probably cannot be a maintainer.

Reply:
Okay
```suggestion
```